client/app/bundles/course/assessment/submission/components/ScribingView/ScribingToolbar.jsx

```diff
@@ -502,15 +505,7 @@ class ScribingToolbar extends Component {
                 scribing.activeObject.type !== 'line') ||
               false
             }
-            iconComponent={() => (
```

Comment:
Missing logic?

Reply:
this one is actually more towards poor naming choice :( I will take note of this matter seriously in the future so that it won't happen again